source/slang/slang-emit.cpp

```diff
@@ -1419,6 +1420,19 @@ Result linkAndOptimizeIR(
     // we will need to disable this pass.
     stripWitnessTables(irModule);
 
+    switch (target)
+    {
+    // On targets that don't support default initialization, remove 'raw' default construct
+    // insts because our code-gen will not have any way to emit them.
+    //
+    case CodeGenTarget::SPIRV:
```

Comment:
What if the raw default constructor has uses? If we just remove them the rest of the IR will be ill-formed.

Reply:
Oh, I see the code is detecting `store(var, defaultConstruct())` and remove the `store` along with the defaultConstruct.